### PR TITLE
Dropdown content UI

### PR DESCRIPTION
**Pull request by panerorenn9541**
- Make heat map type selection look consistent with other map UI features
- Create toggle button with icon for both directions and heat map sections
- Directions section contain entry fields and submit button icon that flow with the rest of the website

### CODE REVIEW

**Comment by panerorenn9541** [comment]
image

Directions drop down. Buttons change color to white when hovered over and clicked

**Comment by panerorenn9541** [comment]
image

Heat map drop down. This section will get bigger in the future, which will expand the drop down content

File: src/main/webapp/routing.js [hunk 1]
```diff
@@ -152,6 +152,30 @@ function showAlternateRoutes() {
   }
 }
 
+/**
+ * Switch alternate routes on and off
+ */
+function toggleAlternateRoutes() {
```
Comment (kinsaurralde) [inline src/main/webapp/routing.js:283]: This function is in #19

**Review by kinsaurralde — APPROVED** [review]

**Comment by panerorenn9541** [comment]
image

With new routing features: add button for expanded information

**Comment by panerorenn9541** [comment]
image

When expanded information button is pressed

**Review by alsile — APPROVED** [review]
awesome changes! Two UI things: 1. the font/text in the drop down menu of the heat map (the one with show confirmed cases etc.)  doesn't look as nice as all the other changes. The font/text looks nice in the direction drop down, the other drop down might look better if you use the same font styling on it as you did with directions. 2. I think bolding all of the text and making it bigger might look better (it was slightly hard to see the text from the pictures).

**Comment by panerorenn9541** [comment]
image

Heat map drop down. More content will be added to this section, which will increase the size of the display

**Comment by panerorenn9541** [comment]
image

Directions drop down

**Comment by panerorenn9541** [comment]
image

Directions with expanded information and alternate routes

**Comment by panerorenn9541** [comment]
image

Increase size of data box

**Comment by panerorenn9541** [comment]
image

Video search bar next to location search. Also increase size of stats box

**Comment by panerorenn9541** [comment]
image

Heat map drop down. More content will be added here, which will increase the size of the display

**Comment by panerorenn9541** [comment]
image

Directions drop down

**Comment by panerorenn9541** [comment]
image

Directions drop down with alternate routes and expanded info

**Review by v01dstar — APPROVED** [review]
LGTM

**Review by huzcn — APPROVED** [review]
Great!

**Pull request merged**